--- grfcodec-r2306/Makefile~	2009-08-04 03:19:35.000000000 +0200
+++ grfcodec-r2306/Makefile	2010-04-22 16:43:52.227800131 +0200
@@ -68,7 +68,7 @@
 endif
 
 # use 386 instructions but optimize for pentium II/III
-CFLAGS = -g -D$(TYPESIZE) -O3 -I. -O1 -idirafter$(BOOST_INCLUDE) -Wall -Wno-uninitialized $(CFLAGAPP)
+CFLAGS = -g -D$(TYPESIZE) -I. -idirafter$(BOOST_INCLUDE) -Wall -Wno-uninitialized $(CFLAGAPP)
 
 ifeq ($(DEBUG),1)
 CFLAGS += -DDEBUG
